bundles/edu.kit.kastel.sdq.eclipse.student.view/src/edu/kit/kastel/eclipse/student/view/ui/ArtemisStudentView.java

```diff
@@ -0,0 +1,325 @@
+import org.eclipse.ui.part.ViewPart;
+import org.eclipse.wb.swt.SWTResourceManager;
+
+import edu.kit.kastel.eclipse.student.view.controllers.StudentViewController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class creates the view elements for the artemis grading process. It is
+ * build as a tab folder with four tabs: grading, assessment, exam and backlog.
+ *
+ * @see {@link ViewPart}
+ *
+ */
+public class ArtemisStudentView extends ViewPart {
+	private static final String NO_SELECTED = "*NOTHING SELECTED*";
+	private List<ArtemisStudentTab> tabs = new ArrayList<>();
+	private StudentViewController viewController;
+	
+	private ScrolledComposite scrolledCompositeGrading;
+	private Composite gradingComposite;
+	
+	private Button btnSubmitExcerise;
+	private Button btnClean;
+	private Combo examCombo;
+	private Combo exerciseCombo;
+	private Combo courseCombo;
+	
+	private ControlDecoration controlDecorationSubmitted;
+	private ControlDecoration controlDecorationClean;
+
+	public ArtemisStudentView() {
+		this.viewController = new StudentViewController();
+		ResultTab resultTab = new ResultTab(viewController);
+		tabs.add(resultTab);
+		tabs.add(new ExamTab(viewController));
+		viewController.connectToWebsocket(resultTab);
+	}
+	/**
+	 * This methods creates the whole view components.
+	 */
+	@Override
+	public void createPartControl(Composite parent) {
+		this.createView(parent);
+	}
+	
+	private void createView(Composite parent) {
+		TabFolder tabFolder = new TabFolder(parent, SWT.NONE);
+		createMainTab(tabFolder);
+		createAllTabs(tabFolder);
+	}
+
+	@Override
+	public void setFocus() {
+		// NOP
+	}
+
+	public void createMainTab(TabFolder tabFolder) {
+		TabItem tbtmAssessment = new TabItem(tabFolder, SWT.NONE);
+		tbtmAssessment.setText("Exercise");
+
+		this.scrolledCompositeGrading = new ScrolledComposite(tabFolder, SWT.BORDER | SWT.H_SCROLL | SWT.V_SCROLL);
+		tbtmAssessment.setControl(this.scrolledCompositeGrading);
+		scrolledCompositeGrading.setLayout(new GridLayout(1, true));
+		this.scrolledCompositeGrading.setExpandHorizontal(true);
+		this.scrolledCompositeGrading.setExpandVertical(true);
+
+		this.gradingComposite = new Composite(this.scrolledCompositeGrading, SWT.NONE);
+		this.scrolledCompositeGrading.setContent(this.gradingComposite);
+		this.scrolledCompositeGrading.setMinSize(this.gradingComposite.computeSize(SWT.DEFAULT, SWT.DEFAULT));
+		gradingComposite.setLayout(new GridLayout(1, true));
+		gradingComposite.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, true, 1, 1));
+
+		Composite assessmentComposite = new Composite(gradingComposite, SWT.BORDER);
+		assessmentComposite.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+		assessmentComposite.setLayout(new GridLayout(2, true));
+
+		Label lblCourse = new Label(assessmentComposite, SWT.NONE);
+		lblCourse.setAlignment(SWT.CENTER);
+		lblCourse.setFont(SWTResourceManager.getFont("Segoe UI", 9, SWT.BOLD));
```

Comment:
Maybe use fonts provided by eclipse.

You should also delete the SWTResourceManager, since eclipse should be able to manage that by itself.